vdb-bench-assembly/plugins/vdb-bench-dataservice/dsTestController.js

```diff
@@ -13,8 +13,8 @@
     function DSTestController($scope, DSSelectionService) {
         var vm = this;
 
-        vm.dsDeployInProgress = false;
-
+        vm.dsDeployInProgress = true;
```

Comment:
Seems like the beginning state should be "vm.dsDeployInProgress = DSSelectionService.isDeploying();".  When the test page is displayed this user will probably have started a deployment and it will be in progress.